packages/react-devtools-shared/src/backend/DevToolsFiberComponentStack.js

```diff
@@ -173,24 +162,6 @@ export function getOwnerStackByFiberInDev(
       case SuspenseListComponent:
         info += describeBuiltInComponentFrame('SuspenseList');
         break;
-      case FunctionComponent:
-      case SimpleMemoComponent:
-      case ClassComponent:
-        if (!workInProgress._debugOwner && info === '') {
-          // Only if we have no other data about the callsite do we add
-          // the component name as the single stack frame.
-          info += describeFunctionComponentFrameWithoutLineNumber(
-            workInProgress.type,
-          );
-        }
-        break;
-      case ForwardRef:
-        if (!workInProgress._debugOwner && info === '') {
-          info += describeFunctionComponentFrameWithoutLineNumber(
-            workInProgress.type.render,
-          );
-        }
-        break;
```

Comment:
Since the actual stack usually has a frame inside the current component, we don't have to add it even if there's no owner since otherwise it'll end up show up twice. I'll do a separate PR to align this with captureOwnerStack() which should also exclude these.